torch/autograd/gradcheck.py

```diff
@@ -1495,8 +1493,12 @@ def gradcheck(
         raise_exception (bool, optional): indicating whether to raise an exception if
             the check fails. The exception gives more information about the
             exact nature of the failure. This is helpful when debugging gradchecks.
-        check_sparse_nnz (bool, optional): if True, gradcheck allows for SparseTensor input,
-            and for any SparseTensor at input, gradcheck will perform check at nnz positions only.
+        check_sparse_nnz (bool, optional): if True, gradcheck allows
```

Comment:
minor formatting nit: put these in double backticks ` ``True`` `, ` ``check_sparse_nnz`` `, ` ``check_sparse_nnz != masked`` `,

Reply:
Fixed.